config/routes.rb

```diff
@@ -27,18 +27,18 @@
     end
 
     constraints(-> { JobseekerApplicationsFeature.enabled? }) do
-      scope path: ":job_id" do
-        resources :job_applications, only: %i[new create]
-      end
-
-      resources :job_applications, only: [] do
+      resources :job_applications, only: %i[index] do
         resources :build, only: %i[show update], controller: "job_applications/build" do
           resources :details, only: %i[new create edit update destroy], controller: "job_applications/details"
         end
         get :review
         post :submit
         resource :feedback, only: %i[create], controller: "job_applications/feedbacks"
       end
+
+      scope path: ":job_id" do
+        resource :job_application, only: %i[new create]
```

Comment:
Why the change to resource (singular) and job_application (singular)?

Reply:
> Use resource for jobseeker job_application new/create routes (prevents clash + makes more sense)
It prevents a clash between the create action for a job application and the new route. And it makes more sense because there's never going to be more than one job application for a particular jobseeker for a particular job

Reply:
My guess: We're in the namespace job_ids within the namespace for jobseekers, and there can only be one job application per job per jobseeker.

Reply:
I should probably do the same for saved jobs as well, but I'll make another micro PR for that